packages/create-sitecore-jss/src/templates/angular/package.json

```diff
@@ -61,7 +61,7 @@
     "@ngx-translate/core": "~15.0.0",
     "@ngx-translate/http-loader": "~8.0.0",
     "@sitecore-jss/sitecore-jss-angular": "~22.0.0",
-    "apollo-angular": "~7.0.2",
+    "apollo-angular": "~5.0.2",
```

Comment:
@sc-addypathania @art-alexeyenko "apollo-angular" 5.0.2 is supposed to be used with Angular 16, this change is not correct. You need to upgrade at least to version 6

Reply:
Hmm you're right. I updated the dependency within the already scaffolded app then it worked fine but new app fails.
Issued another PR.

Reply:
@sc-addypathania Thanks!